selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/factory/TestFactoryInitializer.java

```diff
@@ -177,6 +178,21 @@ public FactoryDto withV(String v) {
       return factoryDto.withV(v);
     }
 
+    @Override
+    public DevfileDto getDevfile() {
```

Comment:
Please, consider adding @Nullable to comment when it may be null. The same for WorkspaceConfig

Reply:
I don't think it's needed for Dto, we use Nullable for guice injections only anyway